main.py

```diff
@@ -1,10 +1,51 @@
 import init_django_orm  # noqa: F401
-
+import json
```

Comment:
optimize your imports

Reply:
Yes, change an order. `json` should be first